element_facemap/facial_behavior_estimation.py

```diff
@@ -152,6 +152,9 @@ class File(dj.Part):
         file_path   : varchar(255)  # filepath of video, relative to root directory
         """
 
+        @classmethod 
+        def insert_video_recording(cls, key, relative_video_paths, recording_id, device_id): 
+              video_recording_insert = {**key, 'recording_id': recording_id, 'device_id': device_id}
```

Comment:
What is this for?

Reply:
I was thinking to include it since the pose estimation task generate depends on the fbe.VideoRecording table being inserted into. For LuLab and my script I had inserted into the VideoRecording table at the same time as inserting into the task. This would not work for the generate class function, so I thought to include this additional classmethod so that in the Jupyter notebook we can separate these steps. Although, we can just include the codeblock for VideoRecording insertion into the jupyter notebook, and this classmethod won't be needed.  Should I change it to this?

Reply:
The above implementation is incomplete - that function does not do anything

I agree on the need and intention of this class method, and this is probably the right place to have this.
But let's not include in this PR - for one it's incomplete, and also let's discuss more on what the right implementation would be

Reply:
Sounds good, I'll remove it for now.